pom.xml

```diff
@@ -14,8 +14,9 @@
 
 	<properties>
 		<awaitability.version>3.1.6</awaitability.version>
+		<gridcapa.starter.minio.adapter.version>1.0.0</gridcapa.starter.minio.adapter.version>
 		<logstash.version>4.9</logstash.version>
-		<task-manager.version>1.4.0</task-manager.version>
+		<task-manager.version>1.12.0</task-manager.version>
```

Comment:
I think we should increment the version of task manager with the new one after release!